packages/core/src/codec/clusterProxy.ts

```diff
@@ -26,7 +26,7 @@ export const ClusterProxyArgs = createBytesCodec({
   },
   unpack(unpackable: Uint8Array): RawClusterProxyArgs {
     const id = blockchain.Byte32.unpack(unpackable.slice(0, 32));
-    const minPayment = Uint8Opt.unpack(unpackable.slice(32, 33));
+    const minPayment = Uint64Opt.unpack(unpackable.slice(32, 40));
     return {
       id,
       minPayment: typeof minPayment === 'number' ? BI.from(minPayment) : void 0,
```

Comment:
The error in the [test workflow](https://github.com/sporeprotocol/spore-sdk/actions/runs/8120070798/job/22196785936?pr=89#step:13:1191) occurred because there is a difference in the return types of `Uint8Opt.unpack()` and `Uint64Opt.unpack()` (both ClusterProxyArgs and MutantArgs shares the same issue):
- Uint8Opt.unpack: returns a `number | undefined`
- Uint64Opt.unpack: returns a `BI | undefined`

To address this issue, the logic for the return values should be updated as follows:
```ts
return {
  id,
  minPayment: typeof BI.isBI(minPayment) ? minPayment : void 0,
};
```